example/demo/src/ui-controls/ui.tsx

```diff
@@ -185,3 +188,19 @@ function ShowToastExample() {
     </YStack>
   )
 }
+
+function HapticsExample() {
+  return (
+    <YStack>
+      <Button onPress={haptics()}>
```

Comment:
can we add buttons to test the rest of the haptic options?

Reply:
Yeah - so I will create button for each variant and group it in sections - one for impact, second one for notifications etc.